app/src/main/java/mozilla/lockbox/view/LockedFragment.kt

```diff
@@ -32,11 +34,21 @@ class LockedFragment : Fragment(), LockedView {
         return inflater.inflate(R.layout.fragment_locked, container, false)
     }
 
+    override fun onPause() {
+        super.onPause()
+        closeKeyboard()
+    }
+
     override val unlockButtonTaps: Observable<Unit>
         get() = view!!.unlockButton.clicks()
 
     override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
         super.onActivityResult(requestCode, resultCode, data)
         _onActivityResult.onNext(Pair(requestCode, resultCode))
     }
+
+    private fun closeKeyboard() {
+        val imm = context?.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
+        imm.hideSoftInputFromWindow(view!!.windowToken, 0)
+    }
```

Comment:
I wonder whether we should put this into the base `Fragment` class and call it from the base `onPause()` method.